### DISCUSSION TITLE
Extracting mutual nearest neighbors

### DISCUSSION THREAD

**Discussion by alejandro-jimenezsanchez** [brianhie/scanorama#102] [Q&A]
Hi,

I am using scanorama through scanpy.

Is it possible to extract the nearest neighbors computed by scanorama?

Thanks

**Comment by brianhie** [comment]
Hi @SNRNS, Scanorama currently doesn't support extracting nearest neighbors. This is the code that implements the approximate nearest neighbors search though: https://github.com/brianhie/scanorama/blob/1c224783e68077b682475bcd4f54571ab72d8f90/scanorama/scanorama.py#L518
